modules/logging-upgrading-clo.adoc

```diff
@@ -22,12 +22,12 @@ To update the {clo} to a new major release version, you must modify the update c
 
 . Click the *Red Hat OpenShift Logging* Operator.
 
-. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link. This link text might be *stable* or *stable-5.y*, depending on your current update channel.
+. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link. This link text might be *stable* or *stable-5.9*, depending on your current update channel.
```

Comment:
You could make this an attribute, like {logging-version}, and then use it here. Then, in the future, you'd only have to update the logging version to update all versions used across the docs.

Reply:
TY. The Logging docs will have to be reorganized, so I very much doubt that this content will survive the docs update.

Reply:
Thank you for the merge!